tests/Support/SupportHelpersTest.php

```diff
@@ -699,20 +699,27 @@ public function testLast()
 
     public function testClassUsesRecursiveShouldReturnTraitsOnParentClasses()
     {
-        $this->assertEquals([
-            'Illuminate\Tests\Support\SupportTestTraitOne' => 'Illuminate\Tests\Support\SupportTestTraitOne',
+        $this->assertTrue([
```

Comment:
Use `assertSame` instead. It checks `strict`.

Reply:
Thanks !